pygeoapi/templates/collections/items/item.html

```diff
@@ -1,11 +1,16 @@
 {% extends "_base.html" %}
 {% set ptitle = data['properties'][data['title_field']] or ("Item {}".format(data['id'])) %}
+{% block desc %}{{ data.get('properties',{}).get('description',{}) | string | truncate(250) }}{% endblock %}
```

Comment:
```
{% block desc %}{{ data.get('properties', {}).get('description',{}) | string | truncate(250) }}{% endblock %}
{% block tags %}{{ data['properties'].get('themes', [{}])[0].get('concepts', []) | join(',') }}{% endblock %}
{% if 'thumbnailUrl' in data['properties'] %}
  {% block thumb %}{{ data['properties']['thumbnailUrl'] }}{% endblock %}
{% endif %}
```

Then again, isn't `thumbnailUrl` a bit specific?

Reply:
Thumbnail url indeed is only relevant for metadata records, probably better to remove it here